### PR TITLE
HAWQ-326. Fix specified HAWQ_RELEASE_VERSION for rpm build.

### PR DESCRIPTION
**Pull request by radarwave**
This fix included:
  1) Fixed HAWQ_RELEASE_VERSION for rpm tarball.
  2) Make sure distclean delete all the rpm artifacts.
  3) Added some log output.

### CODE REVIEW

File: contrib/hawq-package/build_hawq_rpm.sh [hunk 1]
```diff
@@ -71,8 +76,14 @@ rpmbuild --define "_topdir ${RPM_TOP_DIR}" \
          -bb SPECS/hawq.spec
 if [ $? != 0 ]; then
     echo "Build HAWQ rpm package failed, exit..."
-    exit 0
+    exit 1
```
Comment (paul-guo-) [inline contrib/hawq-package/build_hawq_rpm.sh]: Why not "exit $?"

File: contrib/hawq-package/build_hawq_rpm.sh [hunk 2]
```diff
@@ -71,8 +76,14 @@ rpmbuild --define "_topdir ${RPM_TOP_DIR}" \
...
 fi
 
 set +x
 popd > /dev/null
+
+echo "========================================================================="
+echo "Build HAWQ rpm package successfully."
```
Comment (paul-guo-) [inline contrib/hawq-package/build_hawq_rpm.sh]: Maybe we should tell users where the rpm package is?

Comment (radarwave) [inline contrib/hawq-package/build_hawq_rpm.sh]: Good point! Fixed.

**Comment by paul-guo-** [comment]
+1

**Comment by wengyanqing** [comment]
LGTM

**Pull request merged**